workloads/wmt/wmt_jax/workload.py

```diff
@@ -0,0 +1,250 @@
+"""WMT workload implemented in Jax."""
+import functools
+import types
+from typing import Tuple
+
+from . import decode
+from . import input_pipeline
+from . import models
+from . import train
+from flax import linen as nn
+from flax.training import common_utils
+import jax
+import jax.numpy as jnp
+import numpy as np
+import spec
+import tensorflow as tf
+
+CONFIG = types.SimpleNamespace(
+    vocab_path="./wmt_256/sentencepiece_model",
+    vocab_size=32000,
+    max_corpus_chars=10**7,
+    dataset_name="wmt17_translate/de-en",
+    eval_split="test",
+    reverse_translation=True,
+    beam_size=4,
+    num_eval_steps=20,
+    num_predict_steps=-1,
+    learning_rate=0.0625,
+    warmup_steps=1000,
+    label_smoothing=0.1,
+    weight_decay=0.0,
```

Comment:
`learning_rate`, `label_smoothing`, and `weight_decay` should all be removed from the configs, because these will be submission provided tunable hpraams